web/src/components/footer/subcomponents/contact/DiscordIcon.js

```diff
@@ -0,0 +1,21 @@
+import { createSvgIcon } from "@mui/material";
```

Comment:
Can we create a components/common/icons directory to keep this? I'll be adding one for spotify as well

Reply:
Also can you rename the file to be snake case?

Reply:
its muscle memory cuz every frontend repo does uppercamel